source/style-guide/markup-guidelines.txt

```diff
@@ -4,47 +4,61 @@
 Markup Guidelines
 =================
 
+.. include:: /includes/styles/corrections.rst
+
 .. include:: /style-guide/includes/page-needs-update.rst
 
 This section of the style guide provides detailed guidelines about and
-instructions for applying markup conventions for ReStructure Text (RST)
-and MarkDown (MD).
+instructions for applying markup conventions for reStructuredText (RST)
+and Yet Another Markup Language (YAML) files.
 
-ReStructured Text (RST)
-~~~~~~~~~~~~~~~~~~~~~~~
+ReStructured Text
+-----------------
 
-MongoDB documentation uses reStructuredText (RST) markup syntax
-with Sphinx extensions.
+MongoDB documentation uses reStructuredText markup syntax
+with `Sphinx extensions <http://www.sphinx-doc.org/en/master/usage/extensions/index.html>`__.
 
-RST is a powerful and straightforward markup language that, in
-combination with Sphinx, provides a wide range of facilities for
-intelligent and appealing documentation creation. It uses simple
-and implicit syntax to introduce a variety of content elements such
-as titles, code blocks, vertical lists, and many others. All the
-source content formatted using RST is stored in files with the ``.rst``
+|rst| is a powerful and straightforward lightweight markup language
+that, in combination with Sphinx, provides a wide range of facilities
+for intelligent and appealing documentation creation. It uses simple
+and implicit syntax to introduce a variety of content elements such as
+titles, code blocks, vertical lists, and many others. All the source
+content formatted using |rst| is stored in files with the ``.rst``
 extension.
 
-For more information on the RST conventions the Information Development
-team follows, see the following resources:
+For more information on the |rst| conventions the Information
+Development team follows, see the following resources:
 
 * `OpenStack RST conventions <https://docs.openstack.org/contributor-guide/rst-conv.html>`_.
 
-MarkDown (MD)
-~~~~~~~~~~~~~
+Parts of reStructuredText
+-------------------------
 
-MD is a straightforward markup language that uses simple and implicit
-syntax to introduce a variety of content elements. All the source
-content formatted using MD is stored in files with the ``.md``
-extension.
+A :sphinx-rst:`directive </directives.html?highlight=directive>` defines block-level markup.
+
+Docutils provides the following directives
+
+.. code-block:: rst
+
+   .. directive::
+
+   :role:
+
+Parts of a Document
+-------------------
+
+Page
+  Rendered HTML page
```

Comment:
[nit]

I'm not sure I agree with this suggestion. To me, saying "HTML page that renders" introduces ambiguity because it's unclear whether the HTML page is being rendered or doing the rendering.

I find the original wording "Rendered HTML page" clearer.

Reply:
@jeff-allen-mongo, see the new [PR](https://github.com/mongodb/docs-meta/pull/128). I think @ccho-mongodb will close this one.